ansible/worker/hosts

```diff
@@ -0,0 +1,11 @@
+[worker-machine]
+#WORKER-PATTERN-HELPER#
+[worker-machine:vars]
+ansible_ssh_private_key_file=
+ansible_python_interpreter=/usr/bin/python3
+
+[worker-node-machine]
+#WORKER-NODE-PATTERN-HELPER#
+[worker-node-machine:vars]
```

Comment:
The `#WORKER-PATTERN-HELPER#` tag is no longer needed. You can replace it with a blank line.